ui.tests/test-module/specs/embedFormsInSiteOverrideConfig/embedFormsInSiteOverrideConfig.spec.js

```diff
@@ -0,0 +1,45 @@
+/*******************************************************************************
+ * Copyright 2023 Adobe
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ******************************************************************************/
+describe("Form in site using embed container", () => {
+
+    context('Forms inside site using adaptive form embed container', function () {
+  
+      before(() => {
+        cy.attachConsoleErrorSpy();
+      });
+      const pagePath = "/content/forms/sites/core-components-it/aemembedformstest.html";
+      let formContainer = [];
+
+      beforeEach(function () {
+          cy.previewForm(pagePath, {multipleEmbedContainers: true}).then(p => {
```

Comment:
multipleEmbedContainer is set to true, whereas the page only has one embed container.